GoogleDataTransport/GDTCORTests/Unit/GDTCORFlatFileStorageTest.m

```diff
@@ -823,78 +849,88 @@ - (void)testCheckEventExpiration {
   [self waitForExpectations:@[ expectation ] timeout:10];
 }
 
-- (void)testBatchIDsForTarget {
-  __auto_type expectedBatch = [self generateAndBatchEvents];
-
-  XCTestExpectation *batchIDsExpectation = [self expectationWithDescription:@"batchIDsExpectation"];
+- (void)testCheckForExpirations_WhenBatchWithNotExpiredEventsExpires {
+  NSTimeInterval batchExpiresIn = 0.5;
+  // 0.1. Generate and batch events
+  __auto_type generatedBatch = [self generateAndBatchEventsExpiringIn:1000
+                                                      batchExpiringIn:batchExpiresIn];
+  NSNumber *generatedBatchID = [[generatedBatch allKeys] firstObject];
+  NSSet<GDTCOREvent *> *generatedEvents = generatedBatch[generatedBatchID];
+  // 0.2. Wait for batch expiration.
+  [[NSRunLoop currentRunLoop]
+      runUntilDate:[NSDate dateWithTimeIntervalSinceNow:batchExpiresIn + 0.5]];
```

Comment:
Maybe move the 0.5 to a variable?

Reply:
Seems it should work without additional delays, so I removed it.